infrastructure/ArmTemplates/antareskvtemplate.json

```diff
@@ -370,9 +370,19 @@
                   "subStatus": 0,
                   "count": 100,
                   "timeInterval": "00:10:00"
-                }
+                },
+                {
+                  "status": 502,
+                  "subStatus": 0,
```

Comment:
quick question - will this work for 502.3 errors as well, Basically all 502 errors?

Reply:
I just validated by looking at AutoHealing source. If we are setting substatus or win32Status to 0, then it covers status+sub-status scenarios of 502. So the current configuration is accurate and will catch 502.3 and other substatus as well.